web-common/src/features/sources/errors/ErrorPane.svelte

```diff
@@ -7,7 +7,7 @@
   import { useSourceFromYaml } from "../selectors";
 
   export let sourceName: string;
-  export let errorMessage: string;
+  export let errorMessage: string | undefined;
```

Comment:
Can we instead only show the `ErrorPane` if an error message exists?

Reply:
Yep!